src/libawkward/builder/GrowableBuffer.cpp

```diff
@@ -142,4 +146,7 @@ namespace awkward {
   template class EXPORT_TEMPLATE_INST GrowableBuffer<uint64_t>;
   template class EXPORT_TEMPLATE_INST GrowableBuffer<float>;
   template class EXPORT_TEMPLATE_INST GrowableBuffer<double>;
+  template class EXPORT_TEMPLATE_INST GrowableBuffer<std::complex<float>>;
```

Comment:
is it needed?

Reply:
Your Complex128Builder is undoubtedly using the `GrowableBuffer<std::complex<double>>`, but probably nothing is using `GrowableBuffer<std::complex<float>>`. But the same could be said of `GrowableBuffer<double>` and all of the integer types. Even if they're not being used right now, TypedArrayBuilder will need them (to build arrays with an explicit dtype), so we might as well leave them in.